packages/terra-icon/src/icon/IconConsultInstructionsForUse.jsx

```diff
@@ -1,14 +1,21 @@
 /* eslint-disable */
 import React from 'react';
```

Comment:
This file is auto-generated, any manual edits made to it will be overwritten next time [the generate-icon npm script is run](https://github.com/cerner/terra-core/blob/master/packages/terra-icon/package.json#L45). Another way to do this is to detect the ConsultInstructionsForUse icon while generating icons and use a custom template for generating that icon instead of [the default template](https://github.com/cerner/terra-core/blob/d064b2fc0856d5315ab573e68fcf0d228f1ff890/packages/terra-icon/scripts/src/generate-icon/template.txt).

Reply:
@bjankord Just curious, like in what scenario we will be regenerating the existing icons ? like is it during the creation of new icons ?

Reply:
Added new Template for Icons having tooltip and made that template generic for all the theme icons : dc294a9ec3f4f8825ecd45093a01510045db3e4a

Reply:
@ShettyAkarsh When the design team releases a new version of https://github.com/cerner/one-cerner-style-icons, we run a script that pulls in the SVGs and a CSV mapping aliases to the SVGs and use that to generate the icons. 

In that process, we clean out the existing `packages/terra-icon/src/icon` directory and create a new empty `packages/terra-icon/src/icon` directory before writing all of the in all the generated icon files to it.

* https://github.com/cerner/terra-core/blob/master/packages/terra-icon/scripts/src/generate-icon/removeMakeDirectories.js
* https://github.com/cerner/terra-core/blob/master/packages/terra-icon/scripts/src/generate-icon/index.js#L11